src/main.js

```diff
@@ -3,7 +3,12 @@ import './style.css'
 import router from './router'
 import 'bootstrap/dist/css/bootstrap.css'
 import App from './App.vue'
+import VueTheMask from 'vue-the-mask'
```

Comment:
Todo correcto, he tenido que instalar vue-the-mask con npm install. Sin problema,